source/tutorials/packaging-existing-software.md

```diff
@@ -8,26 +8,42 @@ myst:
 (packaging-existing-software)=
 # Packaging existing software with Nix
 
-One of Nix's primary use-cases is in addressing common difficulties encountered while packaging software, like managing dependencies.
+One of Nix's primary use-cases is in addressing common difficulties encountered with compiling software, such as specifying and obtaining dependencies.
```

Comment:
```suggestion
One of Nix's primary use-cases is in addressing common difficulties encountered with packaging software, such as specifying and obtaining dependencies.
```

The reason for this suggestion is to keep in line with line 14 and further down.